app/models/user.rb

```diff
@@ -130,15 +128,15 @@ def enable_notifications_for_admin
   end
 
   def self.notifiable_on_public
-    User.agents.where(notify_on_public: true).reorder('id asc')
+    self.agents.where(notify_on_public: true).reorder('id asc')
   end
 
   def self.notifiable_on_private
-    User.agents.where(notify_on_private: true).reorder('id asc')
+    self.agents.where(notify_on_private: true).reorder('id asc')
   end
 
   def self.notifiable_on_reply
-    User.agents.where(notify_on_reply: true).reorder('id asc')
+    self.agents.where(notify_on_reply: true).reorder('id asc')
```

Comment:
Is it in the style guide to prefer `self` here?  Also, why not just shorten it to 
```
def self.notifiable_on_public
   agents.where(notify_on_public: true).reorder('id asc')
end
```